### ISSUE TITLE
conf_diff.py shows unexpected output on EXOS 22.4

### ISSUE THREAD

**Issue by btclay** [extremenetworks/ExtremeScripting#36]
On 22.4 extra output (see below) is shown:
```
SW-TEST.21 # run script conf_diff.py
Comparing configurations, please wait...

error-correction off
disable mac-lockdown-timeout ports 49
configure mac-lockdown-timeout ports 49 aging-time 15
configure ports 49 far-end-fault-indication off
disable vman cep egress filtering ports 49
configure port 49 dot1p 0
disable snmp traps port-up-down port 50
enable diffserv examination port 50
configure port 50 reflective-relay off
configure port 50 isolation off
disable flow-control tx-pause port 50
disable flow-control rx-pause port 50
configure ports 50 forward-error-correction off
disable mac-lockdown-timeout ports 50
configure mac-lockdown-timeout ports 50 aging-time 15
configure ports 50 far-end-fault-indication off
disable vman cep egress filtering ports 50
configure port 50 dot1p 0

If line starts with '+', the command has been added since last save.
If line starts with '-', the command was present in the last save, and has been deleted.

Config changes:
- configure ports 27 display-string Auth-TEST-MN
+ configure ports 27 display-string Auth-TEST-MN2
Note that this script has cleared the CLI dirty bit. The configuration has not been saved.
SW-TEST.22 #
```

See also: https://community.extremenetworks.com/extreme/topics/conf-diff-py-explanation-of-returning-output

**Comment by btclay** [comment]
This seems to be an issue on the EXOS side of things

When the Python script calls 'clicmd("debug cfgmgr show configuration file {0}".format(selected_config), True)', part of the output of the EXOS command is written to the current session incorrectly, with certain config files.

Issue has been brought to EXOS development.

**Comment by btclay** [comment]
The fix for this issue (with `exsh.clicmd()`) is scoped for EXOS 22.5. This issue is tracked via CR xos0071304.

**Comment by btclay** [comment]
I just confirmed that this is fixed in a pre-GA build of 22.5, so the fix will be available once 22.5 goes GA.
